bin/gobblin-mapreduce.sh

```diff
@@ -20,6 +20,7 @@ function print_usage(){
   echo "  --jars <comma-separated list of job jars>      Job jar(s): if not set, \"$FWDIR_LIB\" is examined"
   echo "  --workdir <job work dir>                       Gobblin's base work directory: if not set, taken from \${GOBBLIN_WORK_DIR}"
   echo "  --projectversion <version>                     Gobblin version to be used. If set, overrides the distribution build version"
+  echo "  --logdir <log dir>                             Gobblin's log directory: if not set, taken from \${GOBBLIN_LOG_DIR}"
```

Comment:
Can you update the description, it seems the behavior is:

1. `--log-dir` is first checked, if set this the directory used
2. `GOBBLIN_LOG_DIR` is checked second, if set this directory is used, logs are written to `currentDir/logs`
3. If neither of the above two options are specified